src/components/Identity/configurePayload.js

```diff
@@ -0,0 +1,39 @@
+import getBrowser from "../../utils/getBrowser";
+import areThirdPartyCookiesSupportedByDefault from "../../utils/areThirdPartyCookiesSupportedByDefault";
+import ecidNamespace from "../../constants/ecidNamespace";
+
+const addEcidToPayload = (payload, ecid) => {
+  payload.addIdentity(ecidNamespace, {
+    id: ecid
+  });
+};
+export default (
+  payload,
+  migration,
+  idMigrationEnabled,
+  thirdPartyCookiesEnabled
+) => {
+  const ecidToMigrate =
+    idMigrationEnabled && migration.getEcidFromLegacyCookies();
```

Comment:
The master branch no longer has `migration.getEcidFromLegacyCookies()`. Now it's `getEcidFromLegacy`, which returns a promise, which will cause race conditions if you're not careful.